src/components/SignUpForm/SignUpForm.tsx

```diff
@@ -171,45 +160,40 @@ const SignUp = () => {
             )}
           </div>
 
-          <div>
-            <UFormLabel title="City" htmlFor="city" />
+          <div className="text-left">
             <UFormInput
-              id="city"
+              title="City"
               name="city"
-              type="text"
+              value={values.city}
+              required
```

Comment:
you don't need to add "required = true" prop since it's by default has true value in your FormInput component